### PR TITLE
Update comment regarding libc_rt_wasm. NFC.

### PR DESCRIPTION
**Pull request by sbc100**
The previous comment refered to asm.js and kind of assumed that
codegen was happening at link time, which these days is only
the case for LTO builds.

### CODE REVIEW

File: tools/system_libs.py [hunk 1]
```diff
@@ -130,13 +130,18 @@ def read_symbols(path):
...
+  # happens during LTO can generate references to new symbols that didn't exist
+  # in the linker inputs themselves.
+  # These symbols are called libcalls in LLVM and are the result of intrinsics
+  # and builtins at the LLVM level.  These libcalls cannot themselves be part
+  # of LTO because once the linker is running the LTO phase new bitcode objects
+  # cannot be added to link.  Another way of putting it: by the time LTO happens
+  # the decision about which bitcode symbols to compile has already been made.
+  # See: https://bugs.llvm.org/show_bug.cgi?id=44353.
+  # To solve this we put all such libcalls in a separate library that, like
+  # compiler-rt, is never compiled LTO/bitcode (see force_object_files in
```
Comment (kripken) [inline tools/system_libs.py:143]: never compiled "to" perhaps? or "from"? or is the meaning otherwise?

Comment (sbc100) [reply tools/system_libs.py]: i went with "as"

File: tools/system_libs.py [hunk 2]
```diff
@@ -115,13 +115,18 @@ def create_lib(libname, inputs):
...
+  # happens during LTO can generate references to new symbols that didn't exist
+  # in the linker inputs themselves.
+  # These symbols are called libcalls in LLVM and are the result of intrinsics
+  # and builtins at the LLVM level.  These libcalls cannot themselves be part
+  # of LTO because once the linker is running the LTO phase new bitcode objects
+  # cannot be added to link.  Another way of putting it: by the time LTO happens
+  # the decision about which bitcode symbols to compile has already been made.
+  # See: https://bugs.llvm.org/show_bug.cgi?id=44353.
+  # To solve this we put all such libcalls in a separate library that, like
+  # compiler-rt, is never compiled as LTO/bitcode (see force_object_files in
```
Comment (kripken) [inline tools/system_libs.py:128]: what does `is never compiled as LTO/bitcode` mean?

is it `is never compiled as bitcode`?

or is it `is never compiled during LTO`?

(Sorry, I'm not familiar enough with LLVM here, so I'm trying to learn how this works from the comment.)

Comment (sbc100) [reply tools/system_libs.py:128]: It means they always get included as real object files and never as LTO object files (AKA bitcode files).  Unlike the all the other libraries here.

Comment (sbc100) [reply tools/system_libs.py:128]: This means they never take part in the LTO process.

Comment (kripken) [reply tools/system_libs.py:128]: I see, thanks!

**Review by kripken — APPROVED** [review]

**Pull request merged**